src/components/claiming/GenerateAddresses.vue

```diff
@@ -0,0 +1,39 @@
+<template>
```

Comment:
Once you have create the wallet, you can't go back. If you have created your wallet but you didn't generate the addresses, we have to let the user continue in the generate addresses step. So, we have to save in the wallet-key-value is the user have generated their addresses or discuss what happen in that case. Can you create an issue for that?